modules/tftp/tftp_plugin.rb

```diff
@@ -9,6 +9,9 @@ class Plugin < ::Proxy::Plugin
                      :verify_server_cert => true
     validate :verify_server_cert, boolean: true
 
+    # Expose automatic iso handling capability
+    capability -> { Proxy::TFTP::Capabilities.has_extraction }
```

Comment:
There isn't a strict need to call a method on a different class. You can just write
```suggestion
    capability -> { which(Proxy::ArchiveExtract.SHELL_COMMAND) ? 'extraction' : nil }
```

Reply:
Done.